boot/environ/lib/misc/rtlcompat.c

```diff
@@ -23,7 +23,11 @@ CHECK_PAGED_CODE_RTL (
 }
 #endif
 
+#ifdef _WIN64
```

Comment:
Wouldn't it be useful to have these definitions in some "global" platform-specific system header, that might possibly be shared with eg. the kernel?

Reply:
No, since this is a huge hack and completely wrong outside of this module. No idea, why someone decided tp use this name, it's aweful. But I'm not here to fix other people's style crimes.